### ISSUE TITLE
None state param in after_hypothesis_control when hypo result is deviated

### ISSUE THREAD

**Issue by dimzak** [chaostoolkit/chaostoolkit-lib#118]
Hi,
Got below error while running `chaostoolkit-opentracing` control but seems like a very generic error for all controls:
```
[2019-05-09 17:28:31 DEBUG] [__init__:354] After-control 'opentracing' failed
    Traceback (most recent call last):
      File "/home/dimitrioszakas/git/dimzak/chaostoolkit-opentracing/.venv/lib/python3.6/site-packages/chaoslib/control/__init__.py", line 345, in apply_controls
        settings=settings)
      File "/home/dimitrioszakas/git/dimzak/chaostoolkit-opentracing/.venv/lib/python3.6/site-packages/chaoslib/control/python.py", line 129, in apply_python_control
        func(context=context, **arguments)
      File "/home/dimitrioszakas/git/dimzak/chaostoolkit-opentracing/.venv/lib/python3.6/site-packages/chaostracing/control.py", line 131, in after_hypothesis_control
        deviated = not state.get("steady_state_met")
    AttributeError: 'NoneType' object has no attribute 'get'
[2019-05-09 17:28:31 CRITICAL] [experiment:239] Steady state probe 'app-must-be-healthy' is not in the given tolerance so failing this experiment
...
...
[2019-05-09 17:28:37 INFO] [experiment:259] Experiment ended with status: deviated
[2019-05-09 17:28:37 INFO] [experiment:263] The steady-state has deviated, a weakness may have been discovered
```

Tested versions:
```
chaostoolkit==1.2.0
chaostoolkit-lib==1.3.0
chaostoolkit-opentracing==0.1.2
```

**Comment by Lawouach** [comment]
Hi @dimzak,

You were right, there was a generic bug here :) 
Thank you very much for the report and sorry for the trouble.

I'll put out a release quickly.

**Comment by Lawouach** [comment]
Released in 1.3.1 :)
